src/Workspaces/CSharp/Portable/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -1340,7 +1340,8 @@ private static int GetNamespaceId(SyntaxList<MemberDeclarationSyntax> members, N
 
                     // nullable rewrite: Nullable<int> -> int?
                     // Don't rewrite in the case where Nullable<int> is part of some qualified name like Nullable<int>.Something
-                    if (!name.IsVar && (symbol.Kind == SymbolKind.NamedType) && !name.IsLeftSideOfQualifiedName())
+                    // Don't rewrite cref="Nullable{T}" to cref="T?" since the ? syntax is not supported within crefs
```

Comment:
This comment is misleading. The `?` syntax is supported in crefs, as long as you are referencing a constructed generic type. For example, the following works fine:

```xml
<see cref="Enumerable.Sum(IEnumerable{decimal?})"/>
```

The problem with `<see cref="Nullable{T}"/>` is you are actually referencing an open generic type. This is exactly the same situation as not being able to use the shorthand syntax for `typeof(Nullable<>)` (which also references the open generic type).